timeless/auth/views.py

```diff
@@ -57,9 +59,6 @@ def logout():
 
 @bp.route("/forgotpassword", methods=("GET", "POST"))
```

Comment:
@paulodamaso As you in it, I think renaming `bp` to `BP` improves the pylint score (as it is a constant)
If you have time left in the issue could you try it?

Reply:
@ammaratef45 Thanks for the suggestion but doing things "as we are in it" it is not a good practice, because it tends to grow up our pull requests, making the REV and ARC jobs harder. I agree with you that we should rename this variable, but it should be done in another issue.